src/main/java/frc/robot/Robot.java

```diff
@@ -29,6 +32,29 @@ public void robotInit() {
 
         coralArmCommand = new CoralArmCommand(coralArm);
         coralArm.setDefaultCommand(coralArmCommand);
+
+        retractAlgaeArm = new RetractAlgaeArm(algaeArm);
+        algaeArm.setDefaultCommand(retractAlgaeArm);
+
+        lowerCoralElevator = new LowerCoralElevator(coralElevator);
+        coralElevator.setDefaultCommand(lowerCoralElevator);
```

Comment:
We might get a situation here where this causes a command-start loop.

What I mean is that `LowerCoralElevator` is a command with a finish condition (when the elevator is lowered). So when it is started it will lower the elevator and then it will end. 

When it ends, it will be re-started because it is the default and will again lower and then finish and then will be re-started ......

Basically, this will be bad. We need a special case to handle this.
One options is to use `Commands.defer` where if the elevator is not lowered, then it runs the lower commands and otherwise does `Commands.none()` (like below). This will avoid the weird loop.